baseline/client/tests/jspsych-memory-field.test.js

```diff
@@ -0,0 +1,55 @@
+import "@adp-psych/jspsych/jspsych.js";
+import "js/jspsych-memory-field.js";
+import { pressKey } from "./utils.js"
+
+describe("jspsych-memory-field.js plugin", () => {
+    it("loads correctly", () => {
+        expect(jsPsych.plugins["memory-field"]).toBeDefined();
+    });
+
+    it("shows stimulus", () => {
+        const stimulus = "const stimulus";
```

Comment:
Minor detail: Using "const stimulus" and "const buttonLabel" is slightly confusing. Maybe just change it to "some stimulus" (or anything else that doesn't suggest that the stimulus/button label text have anything to do with code)?

Reply:
Good point!